pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -86,6 +87,18 @@ message Metadata {
    uint64 total_bytes = 4;
 }
 
+message SchemaObjectsTarget {
+  // IDs are the descriptor IDs of the schema objects in this target.
+  // egs: tables, databases.
+  repeated uint32 ids = 1 [(gogoproto.customname) = "IDs",
+    (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+}
+
+message TenantsTarget {
+  // IDs are the tenant IDs in this target.
```

Comment:
[nit] s/in this target/being protected, reword accordingly

Reply:
done.